src/RPCServer.ts

```diff
@@ -214,10 +209,10 @@ class RPCServer extends EventTarget {
     this.filterSensitive = filterSensitive || rpcUtils.filterSensitive;
   }
 
-  public async destroy(force: boolean = true): Promise<void> {
+  public async stop(force: boolean = true): Promise<void> {
     // Log and dispatch an event before starting the destruction
-    this.logger.info(`Destroying ${this.constructor.name}`);
-    this.dispatchEvent(new events.EventRPCServerDestroy());
+    this.logger.info(`Stopping ${this.constructor.name}`);
+    this.dispatchEvent(new events.EventRPCServerStopping());
```

Comment:
We shouldn't dispatch a separate event here, it's handled by the `async-init` decorator events now.